File: samples/synchronous/simple/birthday_client.pl
use SOAP::Lite service => "file:BirthdayWS.wsdl";

print greet 'Mike', 32;
print "\n";

foreach ('Mike', 'Lisa'){
    my $name = $_;
    if (greeted $name){
        print "I greated $name\n";
    }else{
        print "I did not greet $name\n";
    }
}

File: samples/synchronous/simple/birthday_client.rb
require 'soap/wsdlDriver'

client = SOAP::WSDLDriverFactory.new( "BirthdayWS.wsdl").create_rpc_driver

puts client.message

puts client.greet('Mike', 32)

%w(Mike Lisa).each do |name|
  if client.greeted name
    puts "I greeted #{ name }"
  else
    puts "I did not greet #{ name }"
  end
end



